src/pages/søknad/steg/bo-og-opphold-i-norge/EktefellePartnerSamboer.tsx

```diff
@@ -0,0 +1,156 @@
+import { Checkbox, Input, Radio, RadioGruppe } from 'nav-frontend-skjema';
+import React, { useEffect, useState } from 'react';
+
+import * as personApi from '~api/personApi';
+import { Person } from '~api/personApi';
+import { KjønnKvinne, KjønnMann, KjønnUkjent } from '~assets/Icons';
+import { showName } from '~features/person/personUtils';
+import { Nullable } from '~lib/types';
+
+import { EPSFormData } from './Bo-og-opphold-i-norge';
+import styles from './ektefelle-partner-samboer.module.less';
+import { initialEPS } from './utils';
+
+interface Props {
+    onChange: (eps: EPSFormData) => void;
+    value: Nullable<EPSFormData>;
+}
+const EktefellePartnerSamboer = ({ onChange, value }: Props) => {
```

Comment:
Tror ting kan bli litt finere dersom vi gjør

```suggestion
const EktefellePartnerSamboer = ({ onChange, value = initialEPS }: Props) => {
```

sånn at vi slipper å legge inn `?? initialEPS` på alle spredningene.

Reply:
Ah yes, lurt!